vault-core/src/main/java/org/apache/jackrabbit/vault/fs/io/AbstractExporter.java

```diff
@@ -398,6 +400,28 @@ private static PackageType detectPackageType(WorkspaceFilter filter)  {
         return PackageType.MIXED;
     }
 
+    /**
+     * Returns the version of an Oak module.
+     *
+     * @param moduleName the name of the module
+     * @param clazz a class of the module
+     * @return the version (or "SNAPSHOT" when unknown)
+     */
+    private String getVersion(String moduleName, Class<?> clazz) {
```

Comment:
Can we have a test for this?

Reply:
I have added minimal test coverage; but I'm not really sure whether we actually want to expose the method for testing only.

Is there a test that generated a full package, and where we can inspect the properties.xml later on?